sale_project_fixed_price_task_completed_invoicing/models/sale_order_line.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+from odoo.exceptions import ValidationError
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    @api.constrains('product_id')
+    def _onchange_product_id(self):
+        for line in self:
+            if ('track_service' == line.product_id.track_service and
+               line.product_uom_qty != 1.0):
+                raise ValidationError('Error! You cannot have more than one '
+                                      '"track_service" sold in one Sale Order '
+                                      'Line')
```

Comment:
error message must be translatable

_('The quantity for 'Complete Task' products must be exactly one')